sotodlib/obs_ops/splits.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+from sotodlib.core import FlagManager
+
+def det_splits_relative(aman, det_left_right=False, det_upper_lower=False, det_in_out=False):
+    """
+    Function for adding relative detector splits to aman. A new FlagManager called det_flags will be created and the flags put there.
+
+    Parameters
+    ----------
+    aman : AxisManager
+        Input axis manager.
+    det_left_right: Bool
+        Perform a detector left/right split
+    det_upper_lower: Bool
+        Perform a detector upper/lower split
+    det_in_out: Bool
+        Perform a detector in/out split
+
+    Returns
+    -------
+    aman : AxisManager
+        Updated aman with the new flags
+    """
+
+    try:
+        aman.det_flags
+    except KeyError:
+        aman.wrap('det_flags', FlagManager.for_tod(aman))
```

Comment:
Create a new axis manager called `det_flags` or `dflag_aman` and only wrap it if the wrap argument is not False/None.